src/Tribe/Organizer.php

```diff
@@ -647,12 +648,51 @@ public static function get_fetch_callback( $event ) {
 				)
 			);
 
-			$organizers    = ! empty( $organizer_ids )
-				? array_map( 'tribe_get_organizer', $organizer_ids )
+			$organizers = ! empty( $organizer_ids )
+				? array_map( 'tribe_get_organizer_object', $organizer_ids )
 				: [];
 
 			return array_filter( $organizers );
 		};
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$event" missing

<sub>hash: c99bc1e6642ad11d16717c28b9dbb6e3</sub>